src/pages/AdminDashboard/AdminSettings.tsx

```
@@ -1,11 +1,322 @@
+import LoadingBtn from "../../components/LoadingBtn";
+import Modal from "../../components/Modal";
+import PromptsCard from "../../components/PromptsCard";
+import { FaTimes, FaUserCircle } from "react-icons/fa";
+import checkIcon from "../../assets/images/checkIcon.png"
+import { useNavigate } from "react-router-dom";
 const AdminSettings = () => {
+  const [activeTab, setActiveTab] = useState("details");
+  const [oldPassword,setOldPassword] = useState("")
+  const [newPassword, setNewPassword] = useState("");
+  const [confirmPassword, setConfirmPassword] = useState("");
+  const [feedbackType, setFeedbackType] = useState("")
```

ESLint

[error] 'feedbackType' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)